rules/sflow.mk

```diff
@@ -0,0 +1,45 @@
+# host-sflow package
+
+HSFLOWD_VERSION = 2.0.21
+HSFLOWD_SUBVERSION = 4
+export HSFLOWD_VERSION HSFLOWD_SUBVERSION
+
+HSFLOWD = hsflowd_$(HSFLOWD_VERSION)-$(HSFLOWD_SUBVERSION)_amd64.deb
+$(HSFLOWD)_DEPENDS += $(LIBHIREDIS_DEV) $(LINUX_HEADERS) $(LINUX_HEADERS_COMMON)
+$(HSFLOWD)_SRC_PATH = $(SRC_PATH)/sflow/hsflowd
```

Comment:
where is the dbg package for hsflowd?

Reply:
can you following the instruction here to add dbg package and src code?

https://github.com/Azure/sonic-buildimage/blob/master/README.buildsystem.md#to-enhance-debug-dockers

Reply:
done